renovate.json

```diff
@@ -0,0 +1,24 @@
+{
```

Comment:
don't we need to use renovate only in the main branch ?  please check and remove if needed